src/views/product-downloads-view/components/page-header/index.tsx

```diff
@@ -7,15 +7,25 @@ import { useCurrentVersion } from 'views/product-downloads-view/contexts'
 import { getPageSubtitle } from 'views/product-downloads-view/helpers'
 import s from './page-header.module.css'
 
-const PageHeader = (): ReactElement => {
+interface PageHeaderProps {
+	isEnterpriseMode: boolean
```

Comment:
Should this be optional since you're setting the default value below?

Reply:
I think they have the same effect! I can't remember if TS yells at us for doing both in this instance. I know it would if a function parameter was marked optional and had a default value set.

Reply:
Ah okay cool, if they have the same effect then TIL 😆